Project SF/Views/Tabs/Competitions/CompetitionsView.swift

```diff
@@ -7,7 +7,7 @@
 
 import SwiftUI
 
-struct Competition: Identifiable {
+struct CompetitionStruct: Identifiable {
```

Comment:
Maybe this should be `CompetitionViewModel`?

Reply:
I just removed it cos it wasn't needed anymore lol